etna/models/nn/utils.py

```diff
@@ -310,3 +312,34 @@ def _make_target_prediction(self, ts: TSDataset, horizon: int) -> Tuple[TSDatase
         ts.df = ts.df.iloc[-horizon:]
         ts.loc[:, pd.IndexSlice[:, "target"]] = predicts.T[:horizon]
         return ts, prediction_dataloader
+
+
+class MultiEmbedding(nn.Module):
```

Comment:
Shouldn't we write tests about this class?

Reply:
I will add